### ISSUE TITLE
Lean Publishing

### ISSUE THREAD

**Issue by singularityhacker** [singularityhacker/Content#23]
Lean Publishing App Stack

1.  GitHub repo template
2.  Deploy to Netlify button
    1.  creates a private repo for the user
3.  Uses an SSG (vue.js?)
4.  Formated as a commercial book landing page website
5.  Has writing statistics built-in (UI added to existing github action)
6.  Allows purchasing of the book with crypto using Coinbases API
7.  Coinbase API triggers Netlify function that adjusts netlify role to permit access
8.  Preview vs paid content is distinguished in MD or source repo configuration

tldr: A way better Leanpub. Customizable, git-centric, self-hosted, better UI, disintermediated. 

User requirements:

*   Github account
*   Netlify account
*   Coinbase account

**Comment by singularityhacker** [comment]
*   Calvin's Institutes
*   AA/NA Texts

**Comment by singularityhacker** [comment]
How do I make money on this?
